### ISSUE TITLE
Server side rendering with Angular Universal

### ISSUE THREAD

**Issue by temaptz** [biig-io/ngx-smart-modal#210]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
Console errors while startup with Angular Universal app

**Expected behavior**
Usual server-side rendering application

**Reproduction of the problem**
1) Create app with Angular Universal https://angular.io/guide/universal
2) install ngx-smart-modal as usual
3) Try to run application

**What is the motivation / use case for changing the behavior?**
Possibility to use ngx-smart-modal in applications with server-side rendering (without browser-specific api)

**Please tell us about your environment:**
NodeJS

* **Smart Modal version:** 0.8.x
7.1.1

* **Angular version:** 2.0.x
~7.2.0

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
NodeJS

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Solution:** 
Using browser-specific api: document, window with special wrappers;
Official angular documentation: https://github.com/angular/universal/blob/master/docs/gotchas.md
https://angular.io/api/common/DOCUMENT

**Comment by jelenaDedskin** [comment]
Any update on this? When it will be released?

**Comment by khylias** [comment]
Hi @jelenaDedskin,

we have review the associated PR and when it's done, we'll release it ! 🤘

**Comment by temaptz** [comment]
Hi, i made changes to associated PR and fixed issues.

**Comment by maximelafarie** [comment]
Hi @temaptz, I commented your PR. It would be a pleasure for me to merge it asap it has no more conflicts! 😉 
Regards! 🖖

**Comment by maximelafarie** [comment]
Closed by #253. Thank you very much for support and your efforts! 🖖😉
